### PR TITLE
Add: instructions on creating basic db backups

### PR DESCRIPTION
**Pull request by jordalgo**

### CODE REVIEW

**Comment by jordalgo** [comment]
@joesepi

File: .gitignore [hunk 1]
```diff
@@ -12,8 +12,14 @@ wp-content/uploads
...
+!setup/deploy-exclude-list.txt
 deploy.sh
+!setup/deploy.sh
 gulpfile.js
+!setup/gulpfile.js
 package.json
-
+!setup/package.json
+backup-db.sh
+!setup/backup-db.sh
```
Comment (joesepi) [inline .gitignore:25]: It is my understanding that the bang prefix for gitignore means that you will not ignore those files. I was expecting to see that you had set `setup/` as a gitignore and were making all those other files exceptions. ??

File: setup/README.md [hunk 2]
```diff
@@ -42,11 +42,19 @@ On your server:
 #### Adding more Wordpress Plugins
 Add the plugins to composer.json (search for the [here](https://wpackagist.org)); a few are in there already as examples.
 
-#### Connecting to the MySql Container from Sequel Pro?
+#### Connecting to the MySql Container from Sequel Pro.
```
Comment (joesepi) [inline setup/README.md:45]: Is this specific to Sequel Pro or might there be other SQL tools that could work as well?

File: setup/README.md [hunk 3]
```diff
@@ -42,11 +42,19 @@ On your server:
...
 Make sure you check the actual port as it can be different everytime. `docker ps -a`.
 
+#### Remote DB Syncing with Local
+This technique is very basic at the moment.
+- create a folder on your remote server called "_db_backups"
+- Update the "backup-db.sh" file in your root directory with your mysql info and then run this script: `sh backup-db.sh` on your remote server.
+- Then on your local machine, copy the generated mysql dump files: `scp -P 2222 -r USER_NAME@HOST_ADDRESSS:/absolute/path/to/_db_backups/ .`
+- You can then use these gzipped mysql files locally when you spin up your dev environment.
+- *Note* You can also setup a crontab (example in the setup folder) to run this script on a certain schedule.
+
```
Comment (joesepi) [inline setup/README.md:57]: This is great.

File: setup/README.md [hunk 4]
```diff
@@ -58,5 +66,5 @@ If you don't have a use for a custom theme that has a dev workflow run on NPM an
 
 ## Todo
 - Add details on remote server configuration.
-- Include steps on deploying (via rsync to start).
 - Remove gulp (or make it more agnostic to the setup).
+- Create better/safer way to backup whole site (db, uploads, etc...)
```
Comment (joesepi) [inline setup/README.md:70]: 👍

File: setup/setup.sh [hunk 5]
```diff
@@ -6,6 +6,7 @@ customized_files=(
   'deploy-exclude-list.txt'
   'gulpfile.js'
   'package.json'
+  'backup-db.sh'
```
Comment (joesepi) [inline setup/setup.sh:9]: You do a back up on setup? I guess it makes sense. Just curious.

**Review by joesepi — COMMENTED** [review]
Cool! (with comments)

File: .gitignore (hunk 1 above)
Comment (jordalgo) [reply .gitignore:25]: I want the setup files to not be gitignored for this repo incase someone wants to pull updated ones from the repo. The ones that get copied to the root from setup are gitignored

File: setup/README.md (hunk 2 above)
Comment (jordalgo) [reply setup/README.md:45]: Right now specific to Sequel pro but I'm sure others could also do the job.

File: setup/setup.sh (hunk 5 above)
Comment (jordalgo) [reply setup/setup.sh:9]: No. This just copies this backup shell script into the root for users to customize.

**Comment by jordalgo** [comment]
@joesepi Thanks for the review! 👍

**Pull request merged**